app.js

```diff
@@ -135,6 +135,8 @@ app.head('/api/v1/streets/:street_id', resources.v1.streets.get)
 app.get('/api/v1/streets/:street_id', resources.v1.streets.get)
 app.put('/api/v1/streets/:street_id', resources.v1.streets.put)
 
+app.get('/api/v1/location', resources.v1.location.get)
```

Comment:
I was wondering if we should call this api `geo`. (It's what [Twitter](https://developer.twitter.com/en/docs/geo/places-near-location/api-reference/get-geo-search) and a few other developer APIs do.) They use "location" when talking about the geotag when speaking to a human audience but "geo" as the category of geographic APIs.

Reply:
That sounds good to me!

Reply:
No need to do that!